code/modules/research/designs/AI_module_designs.dm

```diff
@@ -113,7 +113,6 @@
 	name = "Core Module Design (P.A.L.A.D.I.N.)"
 	desc = "Allows for the construction of a P.A.L.A.D.I.N. AI Core Module."
 	id = "paladin_module"
-	build_type = IMPRINTER
```

Comment:
should probably document this in the PR

Reply:
The change had no impact.

`/datum/design/board` parent type had `build_type = IMPRINTER` - This line was just re-defining it to be the same thing. If I didn't remove this, offstation imprinters would not be able to print them and there was no point in re-defining it as `build_type = IMPRINTER | AWAY_IMPRINTER` when `/datum/design/board` is now that.

tl;dr - Re-defined parent build_type and would have caused a bug if I left it in. Removing it has no functionality change beyond what the PR body already states.